sql/11dbFieldTrpWaiverForm.sql

```diff
@@ -0,0 +1,40 @@
+-- Table for the dbFieldTripWaiverForm
+CREATE TABLE dbFieldTrpWaiverForm (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    child_id INT NOT NULL,
+    child_first_name VARCHAR(256),
+    child_last_name VARCHAR(256),
+    gender VARCHAR(6),
+    birth_date DATE,
+    neighborhood VARCHAR(256),
+    school VARCHAR(256),
+    child_address VARCHAR(256),
+    child_city VARCHAR(100),
+    child_state VARCHAR(100),
+    child_zip VARCHAR(10),
+    medical_issues TEXT,
+    foods_to_avoid TEXT,
```

Comment:
Sorry I just noticed this but I'd remove medical_issues and food_to_avoid, since that info will be put into their own tables to keep everything normalized